middlewares/validators/taskRequests.ts

```diff
@@ -0,0 +1,76 @@
+import joi from "joi";
+import { TaskRequestResponse, TaskRequestRequest } from "../../types/taskRequests";
+import { NextFunction } from "express";
+import { REQUEST_TYPE, REQUEST_STATE } from "../../constants/requests";
+import { GITHUB_URL } from "../../constants/urls";
+
+import config from "config";
+import { TASK_REQUEST_TYPE } from "../../constants/taskRequests";
+const githubOrg = config.get("githubApi.org");
+const githubBaseUrl = config.get("githubApi.baseUrl");
+const githubIssuerUrlPattern = new RegExp(`^${githubBaseUrl}/repos/${githubOrg}/.+/issues/\\d+$`);
+const githubIssueHtmlUrlPattern = new RegExp(`^${GITHUB_URL}/${githubOrg}/.+/issues/\\d+$`); // Example: https://github.com/Real-Dev-Squad/website-status/issues/1050
+
+export const createTaskRequestValidator = async (
+    req: TaskRequestRequest,
```

Comment:
In the `createTaskRequestValidator` function, some fields use optional chaining while others don't. This should be standardized based on whether these fields are optional or required.

Reply:
Can you point out line where i have to use optional changing or where I have to **standardized**  ?

Reply:
In the `createTaskRequestValidator` function, it's important to consistently use optional chaining (?.) for optional fields. For example, if description and markdownEnabled are optional, you should ensure they are accessed with optional chaining throughout the code to avoid potential errors. Could you review the function and standardize this?

Reply:
Sometimes, you check if something is there before using it, and sometimes you don't. I think we should always check for optional things. It's like always looking both ways before crossing the street, even if there's no car. If `description` and `markdownEnabled` are not always there, we should use `?. ` to check:

`description?.length // instead of description.length`
`markdownEnabled?.toString() // instead of markdownEnabled.toString()`